securedrop_export/export.py

```diff
@@ -128,7 +128,13 @@ def safe_check_call(self, command, error_message):
         try:
             subprocess.check_call(command)
         except subprocess.CalledProcessError as ex:
-            self.exit_gracefully(msg=error_message, e=ex.output)
+            # ppdc emits warnings which should not be treated as user facing errors
+            if ex.returncode == 0 and \
+               ex.stderr is not None and \
+               ex.stderr.startswith("ppdc: Warning"):
+                logger.info('Encountered warning: {}'.format(ex.output))
```

Comment:
about to test this with my printer, but in the meantime, just pointing out that the indentation is a little short here... i'll add `black` to the project to help as well...